mtp_cashbook/templates/disbursements/search.html

```diff
@@ -111,123 +110,107 @@ <h1 class="heading-xlarge mtp-heading--compact">{% trans 'Payments made' %}</h1>
           {% for disbursement in disbursements %}
             <tr>
               <!-- {{ disbursement.id }} -->
-              <td>
+              <td class="mtp-sender">
                 <span class="visually-hidden">{% trans 'Prisoner' %}:</span>
                 {{ disbursement.prisoner_name }}
-                <hr/>
+                <br><br>
                 <span class="visually-hidden">{% trans 'Prisoner number' %}:</span>
                 <small>{{ disbursement.prisoner_number }}</small>
               </td>
 
-              <td>
-                {{ disbursement.amount|currency:'£' }}
-                <hr/>
-                <span class="visually-hidden">{% trans 'Sent by' %}:</span>
-                <small>{{ disbursement.method|sendingmethod }}</small>
-                <hr>
-                <small>
-                  {% trans 'Description' %}:
-                  {% if disbursement.remittance_description %}
-                    {{ disbursement.remittance_description|hide_long_text }}
-                  {% else %}
-                    <span class="text-secondary">({% trans 'None given' %})</span>
-                  {% endif %}
-                </small>
+              <td class="mtp-amount">
+                {{ disbursement.amount|currency:'£' }}<br>
+                <br>
+                {% if disbursement.method == 'bank_transfer' %}
+                  <small>
+                    <span class="visually-hidden">{% trans 'Recipient account number' %}:</span>
+                      Account {{ disbursement.account_number }}
+                      <br/>
+                    <span class="visually-hidden">{% trans 'Recipient sort code' %}:</span>
+                      Sort code {{ disbursement.sort_code|format_sortcode }}
+                  </small>
+                  <br>
+                  <br>
+                {% else %}
+                  <small>{% trans 'Cheque' %}</small>
+                  <br><br>
+                {% endif %}
+                {% if disbursement.remittance_description %}
+                  <small>{% trans 'Description' %}:</small><br>
```

Comment:
put elements that break the flow (e.g. insert a blank line) on a new line. it makes them easier to see and keeps the diffs clean